docs/administration/vulnerability-scanning/scan-individual-artifact.md

```diff
@@ -69,3 +69,30 @@ When scanning an OCI image index, Harbor will send scan requests for each of the
 
 
 When an OCI image index is successfully scanned, the summary of the vulnerability report for the OCI image index is aggregated from the individual scan results of the the artifacts referenced by the index. The vulnerability report will show both sets of statistics.
+
+## Export Vulnerability Scans
+
+As a project administrator, you are able to export vulnerability scan results of one or more of your projects as CVS and download it from your Harbor instance.
+
+1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+1. Go to **Projects** and select a project.
+1. Select the checkbox for the projects you would like to export.
+1. Select **Action** and then **Export CVE(s)**.
+
+  ![Export CVE](../../../img/export-vulnerabilities.png)
+
+1. In the Export CVE window you can apply any filters you want to the data included in the CVS.
+
+  * **Repositories:** Filters by repository name. Enter multiple repository names in a comma separated list, repositories match a name `repo*`,or ** for all repositories in the projects.
+  * **Tags:** Filters by tags. Enter multiple tags in a comma separated list, tags match a name `tag*`,or ** for all tags in the projects.
+  * **Labels:** Filters by labels. Select multiple labels from the drop down.
+  * **CVE IDs:** Filters by CVE number. Enter comma separated list of CVE IDs.
+
+    ![Export CVE filter options](../../../img/filter-export-vulnerabilities.png)
+
+1. Click **Export**. Harbor will begin creating the CVS file with your exported CVE details. The generated CVE file will be named `csv_file_` and the current date and time formatted as YYYYMMDDHHMM. For example, `csv_file_202207281130.csv` was created on 07/28/2022 at 11:30.
```

Comment:
formatted as YYYYMMDDHHMMSS.  It is now accurate to the second

Reply:
updated